### PR TITLE
Feature/onboarding screen

### PR DESCRIPTION
**Pull request by kszmigiel**

### CODE REVIEW

File: app_motim_fit/lib/ui/onboarding.dart [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                        ),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.all(40.0),
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Center(
+                              child: Image(
+                                image: AssetImage(
```
Comment (maszhin) [inline app_motim_fit/lib/ui/onboarding.dart]: let's try Image.asset

File: app_motim_fit/lib/ui/onboarding.dart [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                  height: 600.0,
+                  child: PageView(
+                    physics: ClampingScrollPhysics(),
+                    controller: _pageController,
+                    onPageChanged: (int page) {
+                      setState(() {
+                        _currentPage = page;
+                      });
+                    },
+                    children: <Widget>[
```
Comment (maszhin) [inline app_motim_fit/lib/ui/onboarding.dart]: Let's a reusable component from children (i.e. OnboardingFeatureWidget)

File: app_motim_fit/lib/ui/onboarding.dart [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        ),
+      ),
+      bottomSheet: Container(
+        height: 68.0,
+        width: double.infinity,
+        color: Color(0xFF242424),
+        child: GestureDetector(
+          onTap: () => Navigator.pushNamed(context, '/splash'),
+          child: Padding(
+            padding: EdgeInsets.only(top: 17.0, left: 74.0),
```
Comment (maszhin) [inline app_motim_fit/lib/ui/onboarding.dart]: Instead of padding, let's use Center() here

File: app_motim_fit/lib/ui/onboarding.dart [hunk 4]
```diff
@@ -0,0 +1,153 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:app_motim_fit/utilities/styles.dart';
+
+
+class OnboardingFeatureWidget extends StatelessWidget {
```
Comment (maszhin) [inline app_motim_fit/lib/ui/onboarding.dart]: 👏

File: app_motim_fit/lib/ui/onboarding.dart (hunk 3 above)
Comment (kszmigiel) [reply app_motim_fit/lib/ui/onboarding.dart]: and what about the arrow icon?

**Pull request merged**